docs/output_files.md

```diff
@@ -0,0 +1,71 @@
+
+| Column | Description |
+| --- | --- |
+| `barcode` | Sample barcode. |
+| `chrom` | Chromosome of amplicon. |
+| `start` | Start position of amplicon. |
+| `end` | End position of amplicon. |
+| `length` | Length of amplicon. |
+| `name` | Name of amplicon. This comes from the fourth column of BED file used when running `nomadic realtime` (e.g. `-b` flag). |
+| `n_reads` | Number of reads mapping to the amplicon. |
+| `mean_cov` | Mean coverage over the amplicon. |
+| `cov_gr100` | Number of positions within the amplicon having >100x coverage. |
+| `per_cov_gr100` | Percentage of positions within the amplicon having >100x coverage. |
+| `total_cov` | Total coverage over amplicon, i.e. mean coverage times length |
+
+
+### `summary.variants.csv`
+The `summary.variants.csv`contains preliminary information about the variants identified in each sample.
+
+
+Each row contains information about the genotype, depth, quality, and within-sample allele frequency (WSAF) of a specific single-nucleotide polymorphism (SNP) in a specific sample. For all samples the same set of SNPs are described. The set of SNPs described includes all SNPs where at *at least one* sample carried the alternative allele. Note this file is only generated when the `nomadic realtime ... --call` flag is used.
+
+| Column | Description |
+| --- | --- |
+| `barcode` | Sample barcode. |
+| `chrom` | Chromosome of SNP. |
+| `pos` | Position of SNP. |
+| `ref` | Reference nucleotide for SNP. |
+| `alt` | Alternative nucleotide for SNP. |
+| `qual` | Variant quality score of SNP. |
+| `mut_type` | Type of mutation caused by SNP, e.g. synonymous or non-synonymous. |
+| `aa_change` | Amino acid change caused by SNP. For synonymous mutations, we still report (e.g. `V380V`). |
+| `aa_pos` | Amino acid number containing the SNP. |
+| `strand` | Strand of gene containing the SNP. |
+| `amplicon` | Name of amplicon containing SNP.  This comes from the fourth column of BED file used when running `nomadic realtime` (e.g. `-b` flag). |
+| `gt` | Called SNP genotype for the sample. Can be reference (`0/0`), heterozygous (`0/1`), homozygous (`1/1`) or failed QC (`./.`). **Note: these are from `bcftools call` and assume a diploid genome.** |
+| `gq` | SNP genotype quality for the sample. Note this is different than variant quality (`qual`) as it refers to the quality of the genotype call, rather than whether or not the site is variable. |
+| `dp` | Sequencing depth over the SNP. Equivalent to coverage. |
+| `wsaf` | Within-sample alternative allele frequency (`ad_alt / (ad_alt + ad_ref)`), where `ad_ref` and `ad_alt` are the depths of the reference and alternative allele. |
+
+
+## Sharing data
+The results folder for an experiment (`results/<expt_name>`) contains all the outputs from *Nomadic*. 
+
+Only the CSV files starting `summary` and the `metadata` folder are required to relaunch the dashboard with the `nomadic dashboard` command (see [Basic Usage](basic.md)).
```

Comment:
While reading this I am wondering if we should make this somehow more obvious from the folder structure

Reply:
This might be worthwhile but I also think the current folder structure is relatively nice.

But it is true: we could put everything for sharing into ONE folder; that would allow people to just grab that folder.

So maybe:
```
/<expt_name>
   /barcodes
   /summaries
      summary.*.csv
      /metadata
      /vcfs
```

Then they could just share the summaries folder. Note this is super breaking for all existing pipelines that assume the folder structure though

I also thought we could write an easy ```nomadic share <expt_name>```command that collects everything and ZIPs. Could have keyword 'all' to do entire workspace. Let's discuss.